### PR TITLE
Artificial Neural network from issue-93 by Arya

### PR DESCRIPTION
**Pull request by AryaPatel1111**
##Issue 93

- Info about Issue

Closes: #[issue number that will be closed through this PR]

#### Artificial neural Network added
I have added an Artificial Neural Network to the cancer prediction notebook and generated the pickle file for it as well. Also, I have updated the rquirements.txt file for the TensorFlow version used by me for the prediction. I was able to achieve more accuracy score as compared to the current Random forest algorithm.

## Checklist:
 to mark a checkbox:-
- [x] My code follows the code style of this project.
-->

- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.

### CODE REVIEW

**Review by utkarsh0702 — CHANGES_REQUESTED** [review]
@AryaPatel1111 The model's accuracy is great and the code seems right but could you please send a separate notebook and create a pr, not edit some else notebook. Please name the model and the notebook such that it shouldn't create a name conflict for us.
Please do so and I will merge the pr.

**Comment by AryaPatel1111** [comment]
@utkarsh0702 Can you please elaborate about how shall I name the notebook and model and under which folder shall I save it?

**Comment by utkarsh0702** [comment]
@AryaPatel1111 keep the model in the Models folder and notebook  in the Notebooks folder, just name the model and notebook such that it won't replace the other notebook and model of Cancer which are already present there.

**Comment by AryaPatel1111** [comment]
In the latest pull request, I have renamed the notebook as well as the model to CancerPrediction_ANN. Please go through it.

**Review by utkarsh0702 — CHANGES_REQUESTED** [review]
@AryaPatel1111 Please remove these duplicate files:
1) Models/cancer-model.pkl
2) Notebooks/CancerPrediction.ipynb

**Comment by AryaPatel1111** [comment]
@utkarsh0702 I have removed the duplicate files mentioned by you. Please go through it.

**Comment by utkarsh0702** [comment]
@AryaPatel1111 You don't have to do it like this. If we merge it, the original Cancer notebook in the Notebooks folder will be deleted.
I am closing this pr. Do one thing, fork our repository again and add your files there and please create a pr again. After creating a pr mesaage me, i will merge the pr and allocate you points.
